### PR TITLE
passphrase change working

### PR DESCRIPTION
**Pull request by chrisnojima**
This PR integrates passphrase changing. This also supports the call to get if you have pgp keys uploaded so we show a warning banner. Please test this on staging w and w/o pgp keys to validate

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/settings/passphrase/index.desktop.js [hunk 1]
```diff
@@ -39,28 +32,17 @@ function UpdatePassphrase (props: Props) {
...
-        style={{marginBottom: globalMargins.medium}}
-      />
+        style={{marginBottom: globalMargins.medium}} />
       <Button
         type='Primary'
         label='Save'
         disabled={!props.canSave}
-        onClick={props.onSave}
-      />
-      <Text
```
Comment (MarcoPolo) [inline shared/settings/passphrase/index.desktop.js]: How's this suppose to work now?

Comment (chrisnojima) [inline shared/settings/passphrase/index.desktop.js]: ???? that line is just below the red..??

Comment (chrisnojima) [inline shared/settings/passphrase/index.desktop.js]: talked to marco, he was actually talking about the forgot password stuff. there is no flow for this anymore as you can change your pass whenever now

**Pull request merged**